packages/express/src/schematics/init/init.ts

```diff
@@ -10,7 +10,7 @@ import {
   nxVersion,
 } from '../../utils/versions';
 import { Schema } from './schema';
-import { setDefaultCollection } from '@nrwl/workspace/src/utils/rules/workspace';
+import { setDefaultCollection } from '@nrwl/workspace';
```

Comment:
You need to do the same thing in every file.

Reply:
Alright I think I have covered every file that imports this.